website/docs/r/logic_app_standard.html.markdown

```diff
@@ -47,6 +47,11 @@ resource "azurerm_logic_app_standard" "example" {
   app_service_plan_id        = azurerm_app_service_plan.example.id
   storage_account_name       = azurerm_storage_account.example.name
   storage_account_access_key = azurerm_storage_account.example.primary_access_key
+  
+  app_settings = {
+    "FUNCTIONS_WORKER_RUNTIME"      = "node"
+    "WEBSITE_NODE_DEFAULT_VERSION" = "<version>"
```

Comment:
Examples here should have working values for these properties.  The latest I believe that is currently supported is`~18`?
```suggestion
    "WEBSITE_NODE_DEFAULT_VERSION" = "~18"
```